### PR TITLE
Sprint#6/docker compose

### PR DESCRIPTION
**Pull request by ManAnRuck**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,12 @@
-FROM node:carbon
+FROM node:latest
 
 WORKDIR /app
 
 COPY package*.json ./
 
-RUN yarn
+RUN yarn install
+#RUN npm install nodemon@latest -g
```
Comment (ulfgebhardt) [inline Dockerfile:8]: remove?

File: Dockerfile [hunk 2]
```diff
@@ -1,14 +1,12 @@
-FROM node:carbon
+FROM node:latest
 
 WORKDIR /app
 
 COPY package*.json ./
```
Comment (ulfgebhardt) [inline Dockerfile:5]: use `COPY . .` ?

**Review by ulfgebhardt — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (ManAnRuck) [reply Dockerfile:8]: was?

File: Dockerfile (hunk 2 above)
Comment (ManAnRuck) [reply Dockerfile:5]: was?

**Pull request merged**